Formula/sngrep.rb

```diff
@@ -28,11 +28,9 @@ def install
     ENV.append_to_cflags "-I#{Formula["ncurses"].opt_include}/ncursesw" if OS.linux?
 
     system "./bootstrap.sh"
-    system "./configure", "--disable-debug",
-                          "--disable-dependency-tracking",
+    system "./configure", *std_configure_args,
                           "--disable-silent-rules",
-                          "--prefix=#{prefix}",
-                          "--with-openssl=#{Formula["openssl@1.1"].opt_prefix}"
+                          "--with-openssl"
```

Comment:
Only supported value is `yes` (default when no `=<ARG>`). Everything else is equivalent to `no` https://github.com/irontec/sngrep/blob/master/configure.ac#L158